eduNEXT/tvm
# feat: Migrate project init command to a clean architecture

Description
========
This PR migrates project init command to a clean architecture (hexagon architecture).

How to test:
=========
1. Use this branch (JDB/environment-manager-migration)
2. Create and activate a virtual environment
3. Run commands:
```bash
tvm project init
tvm project init [name] [version]
```
5. Run ``make test`` to verify the unit tests.
6. Verify the commands without virtual environment
